### PR TITLE
Added kubernetes-on-ubuntu-vmss QuickStart template

### PR DESCRIPTION
**Pull request by bart-jansen**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### CODE REVIEW

File: kubernetes-on-ubuntu-vmss/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+			}
+		},
+		"vmssName": {
+			"type": "string",
+			"metadata": {
+				"description": "The name of your VMSS cluster."
+			}
+		},
+		"location": {
+            "type": "string",
```
Comment (bmoore-msft) [inline kubernetes-on-ubuntu-vmss/azuredeploy.json]: format the JSON (e.g. SHIFT+ALT+F in VS Code)

File: kubernetes-on-ubuntu-vmss/azuredeploy.parameters.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+		"vmssName": {
+			"value": "GEN-UNIQUE"
+		},
+		"adminUsername": {
+			"value": "GEN-UNIQUE"
+		},
+		"adminKey": {
+			"value": "GEN-SSH-PUB-KEY"
+		},
+		"minNodeCount": {
```
Comment (bmoore-msft) [inline kubernetes-on-ubuntu-vmss/azuredeploy.parameters.json]: remove the non-GEN values and add them as defaultValues in the template

File: kubernetes-on-ubuntu-vmss/azuredeploy.json (hunk 1 above)
Comment (bart-jansen) [reply kubernetes-on-ubuntu-vmss/azuredeploy.json]: good catch, fixed

File: kubernetes-on-ubuntu-vmss/azuredeploy.parameters.json (hunk 2 above)
Comment (bart-jansen) [reply kubernetes-on-ubuntu-vmss/azuredeploy.parameters.json]: moved all non-GEN values to template

**Pull request merged**